libs/backend-ddd/src/opportunity/infrastructure/api/brevo/brevoDeal.ts

```diff
@@ -103,8 +104,20 @@ const convertDomainToBrevoDeal = (domainAttributes: OpportunityWithOperatorConta
 }
 
 const convertDomainToBrevoDealUpdate = (domainUpdateAttributes: OpportunityUpdateAttributes): DealUpdateAttributes => {
-  return {
-    envoy: domainUpdateAttributes.sentToOpportunityHub
+  if (domainUpdateAttributes.sentToOpportunityHub) {
+    let deal_stage = BrevoStatus.Transmitted
+    if (Config.BREVO_DEAL_PIPELINE == '65ce1a5e59d96ff2630f06c8') {
+      // test value, already public in app/backend/.env.exemple
+      deal_stage = BrevoStatus.TestTransmitted
+    }
+    return {
+      envoy: domainUpdateAttributes.sentToOpportunityHub,
+      deal_stage: deal_stage
+    }
```

Comment:
suggestion: peut être plus simple ?

```suggestion
    return {
      envoy: domainUpdateAttributes.sentToOpportunityHub,
      deal_stage: Config.BREVO_DEAL_PIPELINE == '65ce1a5e59d96ff2630f06c8' ? BrevoStatus.TestTransmitted : BrevoStatus.Transmitted
    }
```